### ISSUE TITLE
@remotion/gif Doesn't play nicely with remotion's `staticFile`

### ISSUE THREAD

**Issue by jakerr** [remotion-dev/remotion#986]
Using Gif like so:

```jsx
<Gif
  width={216}
  height={216}
  src={staticFile('/logo.gif')}
  fit="fill"/>
```

Fails with a message:

```
Error loading GIF: TypeError: Failed to execute 'fetch' on 'WorkerGlobalScope': Failed to parse URL from /static-ada075bddc8f/logo.gif
```

Even though I can get Chrome to fetch the gif with a `fetch('/static-ada075bddc8f/logo.gif')` in the console and see the correct gif loaded in the Network tab of DevTools.

**Comment by JonnyBurger** [comment]
Thanks for reporting! Solved in 65188e83a74b0ce81f61ba58d4be1200f9b0627a

**Comment by jakerr** [comment]
Awesome, thanks for the quick solve!
